BusinessLayer/JsonMessageBuilder/JsonMessageBuilder.cpp

```diff
@@ -10,9 +11,25 @@
 #include "DataLayer/MpptData/I_MpptData.h"
 #include "DataLayer/LightsData/I_LightsData.h"
 #include "DataLayer/AuxBmsData/I_AuxBmsData.h"
+#include "DataLayer/CcsData/I_CcsData.h"
+#include "DataLayer/CcsData/CcsData.h"
+#include "../CommunicationLayer/PacketChecksumChecker/I_PacketChecksumChecker.h"
+#include <QDebug>
 
+JsonMessageBuilder::JsonMessageBuilder(const I_PacketChecksumChecker& checksumChecker)
+{
+    connect(&checksumChecker, SIGNAL(validDataReceived(QByteArray)),
+            this, SLOT(validData()));
+}
 JsonMessageBuilder::JsonMessageBuilder()
 {
+
+}
+
+void JsonMessageBuilder::validData()
+{
+    //timer_.restart();
```

Comment:
Empty function, does this do anything? Or were the changes to this class things done in testing? Seems like we can revert all the changes done to this file